newsroom/auth/utils.py

```diff
@@ -0,0 +1,57 @@
+import flask
+import werkzeug
+import superdesk
+
+from typing import Optional, TypedDict
+from flask_babel import _
+from superdesk.utc import utcnow
+
+
+class UserData(TypedDict, total=False):
```

Comment:
This might be better located in `newsroom/users/users.py` file, as this is where the `UsersResource` is defined